caddyhttp/httpserver/logger.go

```diff
@@ -171,7 +171,7 @@ selectwriter:
 		}
 	}
 
-	l.Logger = log.New(l.writer, "", 0)
+	l.Logger = log.New(l.writer, "", log.Flags())
```

Comment:
This should be sufficient. I haven't tested it yet.

Reply:
Have you tested this yet?

Reply:
I did in the meanwhile. Please read my comment below.